docs/cli.rst

```diff
@@ -32,30 +32,30 @@ user
 
   docker-compose exec admin flask mailu user --hash_scheme='SHA512-CRYPT' myuser example.net 'password123'
 
-user_import
+user-import
 -----------
 
 primary difference with simple `user` command is that password is being imported as a hash - very useful when migrating users from other systems where only hash is known.
 
 .. code-block:: bash
 
-  docker-compose run --rm admin python manage.py user --hash_scheme='SHA512-CRYPT' myuser example.net '$6$51ebe0cb9f1dab48effa2a0ad8660cb489b445936b9ffd812a0b8f46bca66dd549fea530ce'
+  docker-compose run --rm admin flask mailu user-import --hash_scheme='SHA512-CRYPT' myuser example.net '$6$51ebe0cb9f1dab48effa2a0ad8660cb489b445936b9ffd812a0b8f46bca66dd549fea530ce'
 
-user_delete
+user-delete
 ------------
 
 .. code-block:: bash
 
-  docker-compose exec admin flask mailu user_delete foo@example.net
+  docker-compose exec admin flask mailu user-delete foo@example.net
 
-config_update
+config-update
 -------------
 
 The sole purpose of this command is for importing users/aliases in bulk and synchronizing DB entries with external YAML template:
 
 .. code-block:: bash
 
-  cat mail-config.yml | docker-compose exec admin flask mailu config_update --delete_objects
+  cat mail-config.yml | docker-compose exec admin flask mailu config-update --delete_objects
```

Comment:
In my environment, this requires `-T` option for `docker-compose exec`, see #926.

Also, it seems that `--delete_objects` is not a boolean flag, but requires some arguments:
> Error: --delete_objects option requires an argument